milli/src/update/settings.rs

```diff
@@ -220,6 +230,23 @@ impl<'a, 't, 'u, 'i> Settings<'a, 't, 'u, 'i> {
         Ok(true)
     }
 
+    fn update_distinct_attribute(&mut self) -> anyhow::Result<bool> {
+        match self.distinct_attribute {
+            Setting::Set(ref attr) => {
+                let mut fields_ids_map = self.index.fields_ids_map(self.wtxn)?;
+                fields_ids_map
+                    .insert(attr)
+                    .context("field id limit exceeded")?;
```

Comment:
```suggestion
            Setting::Set(attr) => {
                let mut fields_ids_map = self.index.fields_ids_map(self.wtxn)?;
                fields_ids_map
                    .insert(&attr)
                    .context("field id limit exceeded")?;
```

Reply:
this would not work since it moves attr, the alternative is to borrow `&self.distinct_attribute`, but i find the `ref` more visible than the '&' and clearer on the intent.